### ISSUE TITLE
Migrate older Tables to newer Table API

### ISSUE THREAD

**Issue by jardakotesovec** [pkp/pkp-lib#9744]
**Describe the issue**
With new [submissions listing](https://github.com/pkp/pkp-lib/issues/7495) it was necessary to introduce new Table component with more flexible API.  New table is now covered in storybook.  But there is couple of use cases using previous Table component.

**Solution**
Refactor older tables to use new API, should be relatively straightforward.

Pull requests:
pkp-lib/main: https://github.com/pkp/pkp-lib/pull/10255 (_merged_)
ui-library/main: https://github.com/pkp/ui-library/pull/389 (_merged_)
ojs/main: https://github.com/pkp/ojs/pull/4378 (_merged_)
omp/main: https://github.com/pkp/omp/pull/1656 (_merged_)
ops/main: https://github.com/pkp/ops/pull/737 (_merged_)

Cleanup PRs:
pkp-lib/main: https://github.com/pkp/pkp-lib/pull/10369 (_merged_)
ui-library/main: https://github.com/pkp/ui-library/pull/407 (_merged_)
ojs/main: https://github.com/pkp/ojs/pull/4427 _(submodule updates only, closed)_
omp/main: https://github.com/pkp/omp/pull/1691 _(submodule updates only, closed)_
ops/main: https://github.com/pkp/ops/pull/764 _(submodule updates only, closed)_

**Comment by blesildaramirez** [comment]
Pull requests:
pkp-lib/main: https://github.com/pkp/pkp-lib/pull/10255
ui-library/main: https://github.com/pkp/ui-library/pull/389
ojs/main: https://github.com/pkp/ojs/pull/4378

Submodule updates only:
omp/main: https://github.com/pkp/omp/pull/1656
ops/main: https://github.com/pkp/ops/pull/737

**Comment by blesildaramirez** [comment]
Raised a separate issue to address one more table to be migrated to use new table component: https://github.com/pkp/pkp-lib/issues/10443 [URNPubIdPlugin]
